src/software/simulated_tests/terminating_validation_functions/ball_kicked_validation.h

```diff
@@ -6,11 +6,9 @@
 /**
  * Checks if the robot represented by robot_id has kicked the ball at
  * any point in the test
- * @param robot_id the ID of the robot in question, there must exist a robot
- * for the given robot_id
  * @param angle the angle that we expect the ball to be kicked towards
  * @param world_ptr the world pointer given by the simulator. Gets updated every tick
- * @param yield yields control to the next routine (coroutines)
+ * @param yield yields control to the next routine (coroutines) with error message
  */
-void robotKickedBall(RobotId robot_id, Angle angle, std::shared_ptr<World> world_ptr,
-                     ValidationCoroutine::push_type& yield);
+void ballKicked(Angle angle, std::shared_ptr<World> world_ptr,
```

Comment:
Why are we changing it to ball kicked? Shouldn't we check if a particular robot kicked the ball rather than just ball movement?